Dockerfiles/agent/Dockerfile

```diff
@@ -90,13 +90,13 @@ RUN tar xzf s6.tgz -C / --exclude="./bin" \
   && tar xzf s6.tgz -C /usr ./bin \
   && rm s6.tgz \
 # Prepare for running without root
-# - Create a dd-agent:root user and give it permissions on relevant folders
+# - Create a sts-agent:root user and give it permissions on relevant folders
 # - Remove the /var/run -> /run symlink and create a legit /var/run folder
 # as some docker versions re-create /run from zero at container start
- && adduser --system --no-create-home --disabled-password --ingroup root dd-agent \
+ && adduser --system --no-create-home --disabled-password --ingroup root sts-agent \
  && rm /var/run && mkdir -p /var/run/s6 \
- && chown -R dd-agent:root /etc/datadog-agent/ /etc/s6/ /var/run/s6/ /var/log/datadog/ \
- && chmod g+r,g+w,g+X -R /etc/datadog-agent/ /etc/s6/ /var/run/s6/ /var/log/datadog/ \
+ && chown -R sts-agent:root /etc/stackstate-agent/ /etc/s6/ /var/run/s6/ /var/log/stackstate-agent/ \
```

Comment:
Can we try and be consistent with the naming. I think it would be great if we can have everything `stackstate-agent`

Reply:
Same for the lines above that has `sts-agent`

Reply:
done